traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -59,10 +59,21 @@ func BuildWhereAndOrderBy(parameters map[string]string, queryParamsToSQLCols map
 		if colInfo, ok := queryParamsToSQLCols[orderby]; ok {
 			log.Debugln("orderby column ", colInfo)
 			orderBy += " " + colInfo.Column
+
+			// if orderby is specified and valid, also check for sortOrder
+			if sortOrder, exists := parameters["sortOrder"]; exists {
+				log.Debugln("sortOrder: ", sortOrder)
+				if sortOrder == "desc" {
+					orderBy += " DESC"
+				} else if sortOrder != "asc" {
+					log.Debugln("Incorrect name for sortOrder: ", sortOrder)
+				}
+			}
 		} else {
 			log.Debugln("Incorrect name for orderby: ", orderby)
```

Comment:
I know you didn't write this part but can it say something like "This column is not configured to support orderby: orderby"